exporter/tanzuobservabilityexporter/README.md

```diff
@@ -1,21 +1,30 @@
 # Tanzu Observability (Wavefront) Exporter
 
-This exporter supports sending traces to [Tanzu Observability](https://tanzu.vmware.com/observability).
+This exporter supports sending both traces and metrics to [Tanzu Observability](https://tanzu.vmware.com/observability).
 
 ## Prerequisites
 
 - [Obtain the Tanzu Observability by Wavefront API token.](https://docs.wavefront.com/wavefront_api.html#generating-an-api-token)
 - [Set up and start a Tanzu Observability by Wavefront proxy](https://docs.wavefront.com/proxies_installing.html) and configure it with the API token you obtained.
 - To have the proxy generate [span RED metrics](https://docs.wavefront.com/trace_data_details.html#red-metrics) from trace data, [configure](https://docs.wavefront.com/proxies_configuring.html) the proxy's `customTracingListenerPorts` and use it for the exporter's endpoint.
 
-## Data Conversion
+## Data Conversion for Traces
 
 - Trace IDs and Span IDs are converted to UUIDs. For example, span IDs are left-padded with zeros to fit the correct size.
 - Events are converted to [Span Logs](https://docs.wavefront.com/trace_data_details.html#span-logs).
 - Kind is converted to the `span.kind` tag.
 - Status is converted to `error`, `status.code` and `status.message` tags.
 - TraceState is converted to the `w3c.tracestate` tag.
 
+## Data Conversion for Metrics
+- Gauge metrics are converted to Gauge metrics
```

Comment:
This line is a little confusing. Maybe you could add some specifics on how the data is modified with links to the OTel/Wavefront definitions,  or just remove this completely if there are no "gotchas" in the conversion?

Reply:
Removed the line per your suggestion.